index.d.ts

```diff
@@ -28,6 +28,36 @@ export interface Options {
 	```
 	*/
 	readonly maxDepth?: number;
+
+	/**
+	Indicate whether to use a `toJSON` method if encountered in the object.
+
+	@default true
+
+	@example
+	```js
+	import {serializeError} from 'serialize-error';
+
+	class UnserializableError extends Error {
+		name = 'UnserializableError';
+
+		toJSON() {
+			// Break serialization
+			return {};
```

Comment:
This does not feel like a realistic example. No one would be doing this.

Reply:
Do we really need an example? Just explain the scenarios of when it would be useful.

Reply:
I dropped the example from the readme too. I suppose this could work in case you change your mind:


	```js
	import {serializeError} from 'serialize-error';

	class CustomError extends Error {
		name = 'CustomError';
		constructor(message) {
			this.time = new Date()
		}
		toJSON() {
			return {
				name: 'CustomError'
			};
		}
	}

	const error = new CustomError('🦄');

	console.log(serializeError(error));
	//=> {name: 'CustomError'}

	console.log(serializeError(error, {useToJSON: false}));
	//=> {name: 'CustomError', message: '🦄', time: '1970-01-01T00:00:00.000Z' stack: 'etc'}
	```